nvflare/apis/impl/seq_relay_manager.py

```diff
@@ -77,7 +77,7 @@ def check_task_send(self, client_task: ClientTask, fl_ctx: FLContext) -> TaskChe
 
         # adjust client window
         win_start_idx, win_end_idx = self._determine_window(task)
-        self.logger.debug(f"{win_start_idx=} {win_end_idx=}")
+        self.logger.debug("win_start_idx {}, win_end_idx {}".format(win_start_idx, win_end_idx))
```

Comment:
This will print it out as win_start_idx 2, win_end_idx 3 instead of win_start_idx=2. We should add the = or : in string